components/webrtc/controller/webrtc_controller.jsx

```diff
@@ -16,7 +16,7 @@ import {Constants, UserStatuses, WebrtcActionTypes} from 'utils/constants.jsx';
 import * as Utils from 'utils/utils.jsx';
 import ring from 'images/ring.mp3';
 import ConnectingScreen from 'components/loading_screen.jsx';
-import SearchBox from 'components/search_bar';
```

Comment:
Not related to fix: change to `SearchBar` just to be consistent with component name.

Reply:
Awesome. That always bugged me since I always forgot to grep for the right name.